src/app/project/list/list-item/list-item.component.html

```diff
@@ -2,28 +2,28 @@
     <div class="list-node" *ngFor="let node of list; let first = first; let last = last;">
 
         <!-- button to expand / close node -->
-        <button type="button" mat-icon-button (click)="toggleChildren(node.id)" class="">
+        <button *ngIf="node.children.length !== 0" type="button" mat-icon-button (click)="toggleChildren(node.id)">
```

Comment:
This causes the arrow to be hidden even for project admins. The arrow needs to be shown for project admins so that they can add child node even if one doesn't already exist.

Reply:
Done in https://github.com/dasch-swiss/dsp-app/pull/825/commits/cd871506a5efe3641da19242f9ba711f17d10a68